packages/guides/src/Intersphinx/InventoryLink.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace phpDocumentor\Guides\Intersphinx;
+
+use function htmlspecialchars;
+use function preg_match;
+
+final class InventoryLink
+{
+    private string $project;
+    private string $version;
+    private string $path;
+    private string $title;
+
+    public function __construct(
+        string $project,
+        string $version,
+        string $path,
+        string $title
+    ) {
+        $this->project = htmlspecialchars($project);
```

Comment:
I think you don't need the `htmlspecialchars` because escaping is done by twig by default. Or is there another reason to do this?

Reply:
Ok, lets delegate this to the output then